lib/cli/command.ts

```diff
@@ -56,6 +59,9 @@ export const loadXudInitClient = async (argv: Arguments<any>) => {
   await loadXudConfig(argv);
 
   const certPath = argv.tlscertpath || path.join(argv.xudir, 'tls.cert');
+  if (!fs.existsSync(certPath)) {
```

Comment:
Same here, or better yet create a separate function that attempts to read and return the cert and use the function in both places, so we don't have code duplication.